### PR TITLE
Zowe v2.4 release notes

### PR DESCRIPTION
**Pull request by anaxceron**
Release notes for Zowe 2.4 release

### CODE REVIEW

File: docs/getting-started/release-notes/v2_4_0.md [hunk 1]
```diff
@@ -82,21 +86,16 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
...
+- Updated the `minimatch` and `keytar` dependencies for technical currency.
+- Updated example for the `zowe profiles create zosmf-profile` command. ([#1152](https://github.com/zowe/zowe-cli/issues/1152))
+- Restored info message on daemon startup. ([#1506](https://github.com/zowe/zowe-cli/issues/1506))
+- Updated `ssh2` dependency to fix "Received unexpected packet type" error on SSH commands. ([#1516](https://github.com/zowe/zowe-cli/issues/1516))
 
+#### Zowe CLI Imperative Framework
 
-#### Extensibility API for Zowe Explorer
+- Updated `glob` and `js-yaml` dependencies for technical currency.
+- Updated `diff2html` and `npm-package-arg` dependencies for technical currency.
```
Comment (zFernand0) [inline docs/getting-started/release-notes/v2_4_0.md:97]: Sorry for being picky, but could we combine this two items ?
```- Updated `glob`, `js-yaml`, `diff2html` and `npm-package-arg` dependencies for technical currency.```

File: docs/getting-started/release-notes/v2_4_0.md [hunk 2]
```diff
@@ -82,21 +86,16 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
...
+- Fixed inconsistent behavior of the Config API introduced in the last version. It now skips loading project config layers when project directory is `false` instead of an empty string.
+- Updated the `Config.search` API to skip loading project config layers when project directory is an empty string. ([#883](https://github.com/zowe/imperative/issues/883))
```
Comment (zFernand0) [inline docs/getting-started/release-notes/v2_4_0.md:99]: Also, these two items could be combined
```- Updated the `Config.search` API to skip loading project config layers when project directory is `false`. ([#883](https://github.com/zowe/imperative/issues/883))```

File: docs/getting-started/release-notes/v2_4_0.md [hunk 3]
```diff
@@ -82,21 +86,16 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
...
-#### Db2 Plug-in for Zowe CLI
-
-
-### Zowe Explorer
-
-
-
-
-#### Zowe Explorer Extension for FTP
+- Updated the `minimatch` and `keytar` dependencies for technical currency.
```
Comment (zFernand0) [inline docs/getting-started/release-notes/v2_4_0.md:89]: personal opinion: I believe that dependency updates should always be at the bottom of all lists as they may not impact end users too much (unless there is something seriously broken that we want to highlight with that dependency update).
This comment applies to the imperative framework dependency updates too.

**Review by zFernand0 — APPROVED** [review]
Changes look great! 😋 
I left a few suggestions in case you wanted to consider them 😉

File: docs/getting-started/release-notes/v2_4_0.md (hunk 3 above)
Comment (anaxceron) [reply docs/getting-started/release-notes/v2_4_0.md:89]: sounds like a good practice to me, thanks for letting me know : )

**Pull request merged**